src/interface/src/app/top-bar/top-bar.component.html

```diff
@@ -66,6 +66,10 @@
       {{ 'help_outline' }}
     </mat-icon>
   </a>
+
+  <!--  temp login button until we wrap up new nav-->
+  <a *ngIf="showLogin$ | async" routerLink="/login" class="nav-link">Login</a>
```

Comment:
Did we decide to not hide this behind any of the flags?

Reply:
This does need to be hidden from the login feature flag.

Reply:
it is hidden by the feature flag 

```
  showLogin$ = this.authService.isLoggedIn$.pipe(
    map(
      (loggedIn) => this.featureService.isFeatureEnabled('login') && !loggedIn
    )
  );
  
  ```